src/components/packetdump.rs

```diff
@@ -490,9 +489,9 @@ impl PacketDump {
 
     fn start_loop(&mut self) {
         if self.loop_thread.is_none() {
-            let tx = self.action_tx.clone().unwrap();
-            let interface = self.active_interface.clone().unwrap();
-            let paused = self.dump_paused.clone();
+            let tx = self.action_tx.take().unwrap();
```

Comment:
this cause panic when we're taking a action and while we're trying to call `start-loop` again